src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -13,13 +13,16 @@ public class IntakeCommand extends Command {
   @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
   private final IntakeSubsystem intakeSubsystem;
 
+  private String pos;
+
   /**
    * Creates a new ExampleCommand.
    *
    * @param subsystem The subsystem used by this command.
    */
-  public IntakeCommand(final IntakeSubsystem intakeSubsystem) {
+  public IntakeCommand(final IntakeSubsystem intakeSubsystem, String pos) {
```

Comment:
Let's change this to a Command Factory in the intakesubsystem.

A command factory returns a command that can be used to assign to a trigger.  This avoids the boilerplate code.

Reply:
I'd also recommend using an `enum` to specify the position of the intake, as opposed to a string.